### ISSUE TITLE
Support DDS image preview on FlagTab

### ISSUE THREAD

**Issue by MacTee** [MacTee/KSP-Mod-Admin-aOS#105]
hmm with two extra libs this would work ---> http://freeimage.sourceforge.net/download.html
I'm not sure if its worth it...

**Comment by diomedea** [comment]
True, the usefulness currently appears rather limited, FlagTab alone may not be worth the effort. May become worthwhile in case of further use of DDS images.
